app/models/team.rb

```diff
@@ -314,11 +314,6 @@ def dynamic_search_fields_json_schema
         schema = Dynamic.send(method, self)
         [schema].flatten.each { |subschema| properties[subschema[:id] || type] = subschema }
       end
-      method = "field_sort_json_schema_type_#{type}"
-      if Dynamic.respond_to?(method)
-        sort = Dynamic.send(method, self)
-        properties[:sort][:properties][sort[:id]] = { type: 'array', title: sort[:label], items: { type: 'string', enum: [sort[:asc_label], sort[:desc_label]] } } if sort
-      end
```

Comment:
This should be present if we want to sort by a dynamic field... was it used only by the `deadline` field? If so, please comment that block of code instead of removing, and please add a comment to the top like "Uncomment to allow sorting by a dynamic field"

Reply:
Yes it only used by deadline field so I'll back it commented